packages/go/ecosystem/ecosystem.go

```diff
@@ -0,0 +1,64 @@
+package ecosystem
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/ossf/osv-schema/packages/go/constants"
```

Comment:
nit: should we have these packages at:

`github.com/ossf/osv-schema/bindings/go/...`  instead?

Reply:
I think `bindings` might have been the original name we talked about, and while I don't have a strong preference either way, I think `bindings` might be a bit nicer?

Reply:
@another-rex WDYT? I think it's nice to make it more explicit that these are language bindings, as opposed to arbitrary packages?

Reply:
we need to repeat the word `osv-schema` either way I think, otherwise the package would still be called `go`.

happy to have bindings as an intermediate directory.

Reply:
done